src/processor/AliasProcessor.ts

```diff
@@ -0,0 +1,42 @@
+import fs from 'fs-extra';
+import { ExportableAlias } from '../exportable';
+import { logger } from '../utils/GoFSHLogger';
+
+export class AliasProcessor {
+  static process(aliasFile: string): ExportableAlias[] {
+    const aliases: ExportableAlias[] = [];
+
+    // Load aliases from alias-file option.
+    if (this.isProcessableAliasFile(aliasFile)) {
+      let aliasFileContent;
+      try {
+        aliasFileContent = fs.readFileSync(aliasFile, 'utf8');
+      } catch (e) {
+        logger.warn(`Alias file read failed with error: ${e.message}`);
+      }
+      if (aliasFileContent != null) {
+        const aliasFileContentLines = aliasFileContent.split('\n');
+        for (let i = 0; i < aliasFileContentLines.length; i++) {
+          const aliasFileContentLine = aliasFileContentLines[i].trim();
+          if (aliasFileContentLine.startsWith('Alias:')) {
+            const alias = ExportableAlias.fromFSH(aliasFileContentLine);
+            if (alias != null) {
+              aliases.push(alias);
+            }
+          }
+        }
```

Comment:
An alias definition may be spread across multiple lines, like this:
```
Alias: $myAlias =
http://example.org/myAlias
```
So, splitting on `\n` is not going to be completely correct.

Reply:
Hi Mint, I have made updates per your comments. Please review the changes and confirm whether you are content with the resolutions.
Regarding the handling of multi-line alias definition, I put a basic parser in place, but wonder whether leveraging the antlr grammar in fsh-sushi would have been a better approach. Let me know your thoughts on that as I am not too familiar with antlr lexers. Thanks!